concent_api/common/decorators.py

```diff
@@ -37,14 +41,14 @@
 crash_logger = getLogger('concent.crash')
 
 
-def require_golem_auth_message(view):
+def require_golem_auth_message(view: Callable) -> Callable:
     """
     Decorator for authenticating golem clients
     Unpacks authorization message signed with the key it contains
     proof that the client indeed has the private part of that key
     """
     @wraps(view)
-    def wrapper(request, *args, **kwargs):
+    def wrapper(request: HttpRequest, *args: Any, **kwargs: Any) -> Union[HttpResponse, JsonResponse]:
```

Comment:
this wrapper returns also `view`, so it should be `Union[HttpResponse, JsonResponse, Callable]:`

Reply:
Same as below, it returns what call to `view()` returns, which is `HttpResponse`.